### PR TITLE
Bio1 2011 q3

### PR DESCRIPTION
**Pull request by AG-88301**

### CODE REVIEW

**Comment by macaquedev** [comment]
Latex on the first `$n$-th` is wrong.  @AG-88301

File: solutions/bio1/UpsideDown/index.tsx [hunk 1]
```diff
@@ -0,0 +1,49 @@
+import { SCodeBlock, SText, STitle, SCode } from "components";
+import { brace } from "solutions/util";
+
+export const UpsideDown = () => {
+  return (
+    <>
+      <STitle>Idea</STitle>
+      <SText>
+        Let's first notice the large constraint on $n$. 
+        This means that we can't simply generate all numbers up to the $n-th$ number, as this would be too slow.
```
Comment (macaquedev) [inline solutions/bio1/UpsideDown/index.tsx:10]: `$n$-th` not `$n-th$`

**Review by macaquedev — CHANGES_REQUESTED** [review]

File: solutions/bio1/UpsideDown/index.tsx (hunk 1 above)
Comment (RedMincrafter) [reply solutions/bio1/UpsideDown/index.tsx:10]: ofc how did u not see that that's such a stupid mistake to make I don't even believe it

**Comment by macaquedev** [comment]
@AG-88301 fix merge conflicts as soon as you push please

**Review by macaquedev — APPROVED** [review]
Good work. Your solution is different to mine.

**Pull request merged**